### PR TITLE
Braket bug

### LINKED ISSUE #339: Fix braket KeyError exception
When we try to run 
```
from braket.aws import AwsDevice
from braket.devices import LocalSimulator
from quri_parts.braket.backend import BraketSamplingBackend
from numpy import pi
from quri_parts.circuit import QuantumCircuit
from quri_parts.core.sampling import create_sampler_from_sampling_backend
from quri_parts.circuit.utils.circuit_drawer import draw_circuit

#device = LocalSimulator()
#device = AwsDevice('arn:aws:braket:::device/quantum-simulator/amazon/sv1')
#device = AwsDevice('arn:aws:braket:::device/quantum-simulator/amazon/tn1')
#device = AwsDevice('arn:aws:braket:::device/quantum-simulator/amazon/dm1')
#device = AwsDevice('arn:aws:braket:us-east-1::device/qpu/ionq/Harmony')
device = AwsDevice('arn:aws:braket:us-east-1::device/qpu/ionq/Aria-1')
#device = AwsDevice('arn:aws:braket:us-east-1::device/qpu/ionq/Aria-2')
#device = AwsDevice('arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet')

circuit = QuantumCircuit(2)
circuit.add_H_gate(0)
circuit.add_CNOT_gate(0, 1)
circuit.add_H_gate(0)
draw_circuit(circuit)
backend = BraketSamplingBackend(device, qubit_mapping={0: 0, 1: 1})
sampler = create_sampler_from_sampling_backend(backend)
sampling_result = sampler(circuit, 100)
print(sampling_result)
```
We get 
```
Exception has occurred: KeyError
'braket.ir.jaqcd.program'
  File "/home/drokles/work/quri-parts/packages/braket/quri_parts/braket/backend/transpiler.py", line 30, in __init__
    device_operation = device.properties.dict()["action"][
  File "/home/drokles/work/quri-parts/packages/braket/quri_parts/braket/backend/sampling.py", line 117, in __init__
    [circuit_transpiler, AwsDeviceTranspiler(device)]
  File "/home/drokles/work/quri-parts/reproduce_bug/bug.py", line 23, in <module>
    backend = BraketSamplingBackend(device, qubit_mapping={0: 0, 1: 1})
KeyError: 'braket.ir.jaqcd.program'
```
Error is thrown here https://github.com/QunaSys/quri-parts/blob/9ffe070c6f51fe80a4d05240fde7369b34487fd2/packages/braket/quri_parts/braket/backend/transpiler.py#L30

### PR DESCRIPTION
**Pull request by ThomasenQunasys**
Closes #339 

# Changes:

- Specify action type key based on device name instead of defaulting to `braket.ir.jaqcd.program`
- Update amazon-braket-schemas to allow schema for Garnet device

### CODE REVIEW

**Comment by ThomasenQunasys** [comment]
It turns out that we only have to update `amazon-braket-schemas` to get it to work

File: packages/braket/quri_parts/braket/backend/transpiler.py [hunk 1]
```diff
@@ -26,10 +26,22 @@ class AwsDeviceTranspiler(SequentialTranspiler):
...
+        name_action_type = {
+            "LocalSimulator": "braket.ir.jaqcd.program",
+            "SV1": "braket.ir.jaqcd.program",
+            "TN1": "braket.ir.jaqcd.program",
+            "dm1": "braket.ir.jaqcd.program",
+            "Harmony": "braket.ir.jaqcd.program",
+            "Aria 1": "braket.ir.openqasm.program",
+            "Aria 2": "braket.ir.openqasm.program",
+            "Garnet": "braket.ir.openqasm.program",
+        }
```
Comment (kwkbtr) [inline packages/braket/quri_parts/braket/backend/transpiler.py]: I think having this kind of mapping is fragile to updates on their platform (e.g. addition of new devices).
Another more robust approach may be to check `braket.ir.openqasm.program` first and then `braket.ir.jaqcd.program`.
(Looking at [the current definition of DeviceActionType](https://amazon-braket-schemas-python.readthedocs.io/en/latest/_apidoc/braket.device_schema.device_action_properties.html#braket.device_schema.device_action_properties.DeviceActionType), only those two seem to be compatible with the gate model)

**Review by kwkbtr — COMMENTED** [review]
Thank you for taking care of this!

File: packages/braket/quri_parts/braket/backend/transpiler.py (hunk 1 above)
Comment (ThomasenQunasys) [reply packages/braket/quri_parts/braket/backend/transpiler.py]: Right.. I believe this fixes it!

**Review by kwkbtr — APPROVED** [review]
👍

**Pull request merged**